_wallets/coldcard.md

```diff
@@ -0,0 +1,26 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: coldcard
+title: "Coldcard"
+titleshort: "Coldcard"
+compat: "hardware"
+level: 2
+platform:
+  - hardware:
+    name: hardware
+    os:
+      - name: hardware
+        text: "walletcoldcard"
+        link: "https://coldcardwallet.com/"
+        source: "https://github.com/coldcard/"
+        screenshot: "coldcard.png"
+        check:
+          control: "checkgoodcontrolfull"
+          validation: "checkneutralvalidationvariable"
+          transparency: "checkgoodtransparencydeterministic"
```

Comment:
I think this should be `checkpasstransparencyopensourcehardware`. I could not find any documentation that mentions that the firmware is deterministically built or any way to verify that the firmware on the device matches what you could build yourself.

Reply:
👍